thriftpy/parser/parser.py

```diff
@@ -10,17 +10,7 @@
 from ply import yacc
 
 from .lexer import tokens, lexer  # noqa
-from .model import (
-    BASE_TYPE_MAPS,
-    Field,
-    Function,
-    IdentifierValue,
-    ListType,
-    MapType,
-    Service,
-    SetType,
-    Thrift
-)
+from .types import *  # noqa
```

Comment:
Please don't do this. It's much easier to reason about the code with explicit imports.